### PR TITLE
Add travis ci documentation

### PR DESCRIPTION
**Pull request by nicolasarnold12321**

### CODE REVIEW

**Comment by nicolasarnold12321** [comment]
https://deploy-preview-459--roadie.netlify.app/docs/integrations/travis-ci/

File: content/docs/integrations/travis-ci/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+### Step 2: Store the token in Roadie
+Visit
+```text
+https://<tenant-name>.roadie.so/administration/settings/secrets
+``` and enter the token value from above into `TRAVISCI_AUTH_TOKEN`.
+
+
+## References
+
+- [Token, Token, Token](https://blog.travis-ci.com/2013-01-28-token-token-token)
```
Comment (martina-if) [inline content/docs/integrations/travis-ci/index.md:39]: This part is not rendering properly. I think you need a new line after line 31 and on line 43 after the <pre>```</pre>


travis

**Comment by nicolasarnold12321** [comment]
@dtuite I have never come across the CISwitcher. Could you link me some documentation to it? I'll add resources for the annotations so.

**Comment by dtuite** [comment]
@nicolasarnold12321 I might be calling it the wrong thing.

If I read these docs, I would understand how to configure the plugin, but I wouldn't know where to go to actually see it in Roadie. I think we should tell the reader how to accomplish that. That's the next thing they're going to want to do.

**Comment by nicolasarnold12321** [comment]
Ah ok. I'll see what I can do!

**Comment by nicolasarnold12321** [comment]
@dtuite I added some resources to the documentation there

**Review by dtuite — APPROVED** [review]

**Pull request merged**